### ISSUE TITLE
.simulate() should provide a basic event object methods by default

### ISSUE THREAD

**Issue by tarjei** [airbnb/enzyme#277]
Hi,
If I do simulate('click') then it would be great if the call to the handler included a bare bones event object containing the normal preventDefault() methods instead of having to define those every time.

**Comment by gmcleod-buzz** [comment]
I second this feature, would be amazing!

**Comment by Cleod9** [comment]
I second this feature, would be amazing!

**Comment by lelandrichardson** [comment]
@tarjei I agree this would be a great addition. I would encourage you to open a PR if this is something that you're really wanting!

I'd like to improve the event simulation for enzyme, but have not yet really gotten around to it in earnest. Any help would be appreciated!
